### DISCUSSION TITLE
Any plans to dockerize vllm?

### DISCUSSION THREAD

**Discussion by tjiahen** [vllm-project/vllm#224] [Q&A]
Any plans to include dockerfile and build process to build the docker image?
I think it will be easier for ppl to get onboard.

**Comment by andreapiso** [comment]
As long as it can be done with pip install, why do you think docker would be easier? I'd say I'd be more cumbersome, especially for enterprise environments where data scientists are not allowed to run arbitrary dockers. Having wheels with precompiled cuda kernels would be perfect.

**Reply by Terristen** [reply]
The answer is because not all corporate environments are the same. I can send a docker container to my network engineer to spin up but cannot do the same with a pip install.

**Reply by andreapiso** [reply]
I don't really understand this point. You could do this simple pip install in a docker file and you'd have your container?  99% of python libraries do not release dockers, but you can build a container that has them?

**Comment by samos123** [comment]
I do agree that ideally this is just part of the main repo, but for now I published a Docker image and shared it here: https://github.com/substratusai/vllm-docker
